cherryontech/bloom-with-us
# Updated nav bar design

# Description

Updated colors and logo as per design. Enabled better accessibility by using draftui buttons


## Type of change

Please delete options that are not relevant.



# Checklist:




# Testing Instructions
Use the deploy link and open a screen reader to check the nav bar, or use your mouse to click through the nav bar options.  



# Gotchas/What I learned
